src/lib/parser/README.md

```diff
@@ -58,82 +52,133 @@ src/lib/parser/
 ├── games/                   # Game-specific configurations
 │   ├── registry.ts          # Available game configurations
 │   ├── quetzal/            # Pokemon Quetzal support
-│   │   ├── config.ts       # Quetzal configuration and Pokemon data
-│   │   └── data/           # Quetzal mappings (pokemon, items, moves)
 │   └── vanilla/            # Pokemon Emerald support
-│       └── config.ts       # Vanilla configuration and Pokemon data
 ├── data/                   # Shared data files
-│   └── pokemon_charmap.json
 └── __tests__/              # Comprehensive test suite
 ```
 
-## Adding New Game Support
+## Supported Games
 
-1. **Create a new game configuration:**
+- **Pokemon Quetzal** - Full support with unencrypted IVs and custom shiny logic
+- **Pokemon Emerald (Vanilla)** - Basic support with encrypted data handling
+
+## Adding Game Support
+
+The parser uses a flexible GameConfig system. Here's how to add support for new Pokemon games:
+
+### Step 1: Create Configuration Class
 
 ```typescript
 export class MyGameConfig implements GameConfig {
   readonly name = 'My Pokemon Game'
   readonly signature = 0x12345678
   
   readonly offsets = {
-    // Define all memory offsets...
+    // Define all memory offsets for your game
+    partyStartOffset: 0x500,
+    partyPokemonSize: 104,
+    maxPartySize: 6,
+    // ... more offsets
   }
   
   readonly mappings = {
-    // Define ID mappings...
+    // Define ID mappings for your game
+    pokemon: this.createPokemonMap(),
+    items: this.createItemMap(),
```

Comment:
simplify this whole adding game support part. it can just be comments and rough structures that need to be done. no need for example offsets or code or anything like that